.github/workflows/lib-injection.yml

```diff
@@ -26,66 +26,56 @@ jobs:
       id: buildx
       uses: docker/setup-buildx-action@v2
 
-    - name: Available platforms
-      run: echo ${{ steps.buildx.outputs.platforms }}
-
-    - name: Set Docker Image Tag
-      id: set_names
+    - name: Copy python tracer
```

Comment:
I think we should keep all the `build-and-publish-init-image` job code here. This is production code that is responsible for building and publishing the container image. Having the code split between here and system-tests seems like unnecessary coupling and is particularly odd as now we'd be introducing production code to the system-tests repository.

Reply:
Ok, Kyle. I agree with you that system-tests shouldn't take part on the "init-image" generation, i think it is a duty of each team and it is important keep in mind, that these images become public releases.


We have only intervened in two things:

- In dd-trace-java they used shell scripts to create the images, and we changed it to GitHub Actions, similar to how it was in python.
- To run the tests in system-tests we need that the images uploaded to the docker registry (docker hub or Github registry) are always taged the same way. For production we were already using "LATEST". We have added that in the case of the build of the main branch, create a tag "LATEST_SNAPSHOT" (see step lib-injection-tags). Probably the shared Github action that calculates the tags to generate, should be in another repository different from system-tests, but always thinking that we should share this action among all tracer repositories. We can move it later.

In the case of python we did modify the Docker file because when we started with this task, the python image was always being created with the ddTrace release, not with the snapshot version of the branch. 

I see that with the changes you have made this week, I would now take the built snapshot version. Is this correct? @Kyle-Verhoog 

Sorry. I should have asked before modifying it.... Now I'm going to undo the changes in the DockerFile and sitecustomize